tests/unit/test_rg_matrix.cpp

```diff
@@ -598,7 +598,7 @@ TEST_F(RGMatrixTest, RGMatrix_managed_transposed) {
 }
 
 // nvals(A eWiseMult B) == nvals(A) == nvals(B)
-void Compare_Matrices(GrB_Matrix m, GrB_Matrix n, GrB_Type t, GrB_Index nrows, GrB_Index ncols)
+void ASSERT_GrB_Matrices_EQ(GrB_Matrix m, GrB_Matrix n, GrB_Type t, GrB_Index nrows, GrB_Index ncols)
```

Comment:
Please change sig to: `void ASSERT_GrB_Matrices_EQ(const GrB_Matrix A, const  GrB_Matrix B)`